src/shared/ui/components/tabs/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Tabs as MuiTabs, Tab, Box } from "@mui/material";
+import clsx from "clsx";
+import React, { Fragment, useCallback } from "react";
+
+interface Props {
+    titles: string[] | JSX.Element[];
+    content: React.ReactNode[];
+    fontWeight?: number;
+}
+
+export const Tabs = ({ titles, content }: Props) => {
+    const [activeТаbIndex, activeTabSwitch] = React.useState(0);
+
+    const onChange = useCallback(
+        (_event: React.ChangeEvent<any>, value: any) => activeTabSwitch(value),
+        [activeTabSwitch]
+    );
+
+    return (
+        <Box sx={{ width: "100%" }}>
+            <Box>
```

Comment:
unnecessary `<Box>`. Also please use divs. To my understanding the only benefit of Box is that is works with sx props. But we dont have/want CSS in JS.